ocl-icd/ocl-icd.spec

```diff
@@ -0,0 +1,55 @@
+Name:                ocl-icd
+Version:             2.2.12
```

Comment:
Any reason why 2.2.12 ?
At https://github.com/bio-arm/bio-arm.github.io/issues/2#issuecomment-1582139940 we saw that Ubuntu uses 2.2.14, it is the last one from the 2.2.x series.
I think it would be good to use 2.2.14 since it has some improvements over 2.2.12.

There is also [2.3.2](https://github.com/OCL-dev/ocl-icd/releases) but it might be incompatible with 2.2.x (used by Bioconductor on Ubuntu)